step/src/main/java/com/google/sps/servlets/CalendarServlet.java

```diff
@@ -0,0 +1,83 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import java.util.Arrays;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import com.google.appengine.api.users.UserServiceFactory;
+import com.google.sps.CalendarManager;
+import com.google.sps.User;
+
+@WebServlet("/cal")
+public class CalendarServlet extends HttpServlet {
+
+  @Override
+  public void init() {
```

Comment:
If there is no body it should just be a one liner like this: `public void init() {}` If there is supposed to be an implementation then add a TODO to implement.